src/Hw.java

```diff
@@ -0,0 +1,32 @@
+import java.util.*;
+
+public class Hw {
+    public static void main(String[] args) {
+        int[] array1 = {5, 11, 7, 20};
+
+        for (int i = 0; i < array1.length; i++) {
+            System.out.println(Arrays.stream(array1).min());
```

Comment:
Incorrect implementation

Reply:
public static void printMinAbsoluteDifference(int[] array) {

        int minDifference = Math.abs(array[0] - array[1]);

        for (int i = 1; i < array.length - 1; i++) {
            int difference = Math.abs(array[i] - array[i + 1]);
            if (difference < minDifference) {
                minDifference = difference;
            }
        }

        System.out.println("The smallest value of the difference between numbers modulo: " + minDifference);
    }